### PR TITLE
Upgrade to Littlefs rev2.4.1 

### PR DESCRIPTION
**Pull request by poojamg**

### CODE REVIEW

File: .github/workflows/post-release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: post-release
+on:
```
Comment (dleach02) [inline .github/workflows/post-release.yml]: Not sure you intended to include these .github files

**Comment by dleach02** [comment]
You will need a PR in Zephyr updating the west.yml

File: .github/workflows/post-release.yml (hunk 1 above)
Comment (poojamg) [reply .github/workflows/post-release.yml]: My bad have removed .github files ..

**Comment by poojamg** [comment]
> You will need a PR in Zephyr updating the west.yml
Thanks @dleach02, I have added it now
https://github.com/zephyrproject-rtos/zephyr/pull/40484

**Review by de-nordic — APPROVED** [review]

**Comment by tejlmand** [comment]
@poojamg this being a merge commit bringing Zephyr fork up-to-data with v2.4.1, then why does the Zephyr fork still contain:
https://github.com/zephyrproject-rtos/littlefs/blob/zephyr/.travis.yml

which were removed here:
https://github.com/zephyrproject-rtos/littlefs/commit/6592719d28d8f3af931680cc04f77cf855a0b3b8

Wrong decision when resolving conflicts ?

**Comment by de-nordic** [comment]
> @poojamg this being a merge commit bringing Zephyr fork up-to-data with v2.4.1, then why does the Zephyr fork still contain: https://github.com/zephyrproject-rtos/littlefs/blob/zephyr/.travis.yml
> 
> which were removed here: [6592719](https://github.com/zephyrproject-rtos/littlefs/commit/6592719d28d8f3af931680cc04f77cf855a0b3b8)
> 
> Wrong decision when resolving conflicts ?

The file does not show on the diff of this PR so I do not know where it comes from.

**Comment by poojamg** [comment]
> > @poojamg this being a merge commit bringing Zephyr fork up-to-data with v2.4.1, then why does the Zephyr fork still contain: https://github.com/zephyrproject-rtos/littlefs/blob/zephyr/.travis.yml
> > which were removed here: [6592719](https://github.com/zephyrproject-rtos/littlefs/commit/6592719d28d8f3af931680cc04f77cf855a0b3b8)
> > Wrong decision when resolving conflicts ?
> 
> The file does not show on the diff of this PR so I do not know where it comes from.

Hmm, I don't see which commit added back this file, I don't understand one commits delete and succeeding commit modifies the file (I don't see adding it back commit) 
image

**Comment by tejlmand** [comment]
> The file does not show on the diff of this PR so I do not know where it comes from.

It does, try fetch this PR and diff it against `v2.4.1` for example:
```
$ git fetch zephyr pull/7/head
$ git diff v2.4.1 FETCH_HEAD -- .travis
diff --git a/.travis.yml b/.travis.yml
new file mode 100644
index 0000000..78d964a
--- /dev/null
+++ b/.travis.yml
@@ -0,0 +1,429 @@
+# environment variables
+env:
+  global:
+    - CFLAGS=-Werror
+    - MAKEFLAGS=-j
+
+# cache installation dirs
+cache:
+  pip: true
+  directories:
+    - $HOME/.cache/apt
+
+# common installation
....
```

I fixed it here: https://github.com/zephyrproject-rtos/littlefs/pull/9

**Comment by tejlmand** [comment]
>  I don't understand one commits delete and succeeding commit modifies the file 

Exactly, so because one branch removes the file and the other modifies it you should have gotten a merge conflict.
And `git` should have asked during conflict resolving whether you want to keep the deleted or modified file.

This is why I wrote: `Wrong decision when resolving conflicts ?` earlier.

**Pull request merged**